geode-core/src/integrationTest/java/org/apache/geode/internal/jta/functional/CacheJUnitTest.java

```diff
@@ -245,7 +246,8 @@ public void testScenario3() throws Exception {
       jtaObj.getRegionFromCache("region1");
 
       String current_fullpath = jtaObj.currRegion.getFullPath();
-      assertEquals("failed retrieving current region fullpath", "/" + DEFAULT_RGN + "/region1",
+      assertEquals("failed retrieving current region fullpath",
+          SEPARATOR + DEFAULT_RGN + "/region1",
```

Comment:
Did you miss this one?, shouldn't it be `SEPARATOR + DEFAULT_RGN + SEPARATOR + "region1"`?